deeplake/core/dataset/dataset.py

```diff
@@ -2931,6 +2929,99 @@ def append(
                             ) from e2
                     raise e
 
+    def extend(
+        self,
+        samples: Dict[str, Any],
+        skip_ok: bool = False,
+        append_empty: bool = False,
+        ignore_errors: bool = False,
+    ):
+        """Appends multiple rows of samples to mutliple tensors at once. This method expects all tensors being updated to be of the same length.
+
+        Args:
+            samples (Dict[str, Any]): Dictionary with tensor names as keys and samples as values.
+            skip_ok (bool): Skip tensors not in ``samples`` if set to True.
+            append_empty (bool): Append empty samples to tensors not specified in ``sample`` if set to ``True``. If True, ``skip_ok`` is ignored.
+            ignore_errors (bool): Ignore errors while appending samples if set to ``True``.
```

Comment:
@farizrahman4u Pls change this to. 

Skip samples that cause errors while appending samples, if set to ``True``.